view.go

```diff
@@ -52,7 +63,20 @@ func (v *View) head(shouldAlter bool) string {
 		ddlOperation = "ALTER"
 	}
 
-	return fmt.Sprintf("%s ALGORITHM=%s DEFINER=%s %s VIEW %s AS ", ddlOperation, v.Algorithm, definer, characteristics, EscapeIdentifier(v.Name))
+	columnNames := make([]string, 0)
```

Comment:
Super nit: I like `var columnNames []string` for the "don't allocate anything" slice declaration since it becomes very obvious that you're going to let Go do its thing and allocate as necessary via append.